data.sql

```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018/11/15', 2, true, 8);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021/01/07', 1, false, 15.04);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017/05/12', 5, true, 11);
+
+/* DAY 2 */
+ALTER TABLE animals ADD COLUMN species text;
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', '2020/02/08', 0, false, -11);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantnom', '2021/11/15', 2, true, -5.7);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (7, 'Squirtle', '1993/04/02', 3, false, -12.13);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (8, 'Angemon', '2005/06/12', 1, true, -45);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (9, 'Boarmon', '2005/06/07', 7, true, 20.4);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Blossom', '1998/10/13', 4, true, 22);
```

Comment:
- [ ] The _**weight_kg**_ and _**escape_attempt**_ detail for _**Blossom**_ is incorrect. It should be `17.0` and `3` respectively. Kindly update it. It is important to input correct data entries so that your queries return the correct results. 👍